example/ffmpeg.js

```diff
@@ -34,47 +35,62 @@ const progressbar = setInterval(() => {
   readline.cursorTo(process.stdout, 0);
   const toMB = i => (i / 1024 / 1024).toFixed(2);
 
-  process.stdout.write(`Audio | ${(tracker.audio.downloaded / tracker.audio.total * 100).toFixed(2)}% processed `);
+  process.stdout.write(`Audio  | ${(tracker.audio.downloaded / tracker.audio.total * 100).toFixed(2)}% processed `);
   process.stdout.write(`(${toMB(tracker.audio.downloaded)}MB of ${toMB(tracker.audio.total)}MB).${' '.repeat(10)}\n`);
 
-  process.stdout.write(`Video | ${(tracker.video.downloaded / tracker.video.total * 100).toFixed(2)}% processed `);
+  process.stdout.write(`Video  | ${(tracker.video.downloaded / tracker.video.total * 100).toFixed(2)}% processed `);
   process.stdout.write(`(${toMB(tracker.video.downloaded)}MB of ${toMB(tracker.video.total)}MB).${' '.repeat(10)}\n`);
 
+  process.stdout.write(`Merged | processing frame ${tracker.merged.frame} `);
+  process.stdout.write(`(at ${tracker.merged.fps} fps => ${tracker.merged.speed})\n`);
+
   process.stdout.write(`running for: ${((Date.now() - tracker.start) / 1000 / 60).toFixed(2)} Minutes.`);
-  readline.moveCursor(process.stdout, 0, -2);
+  readline.moveCursor(process.stdout, 0, -3);
 }, 1000);
 
 // Start the ffmpeg child process
 const ffmpegProcess = cp.spawn(ffmpeg, [
   // Remove ffmpeg's console spamming
   '-loglevel', '0', '-hide_banner',
+  // Redirect/enable progress messages
+  '-progress', 'pipe:3',
   // 3 second audio offset
-  '-itsoffset', '3.0', '-i', 'pipe:3',
-  '-i', 'pipe:4',
+  '-itsoffset', '3.0', '-i', 'pipe:4',
+  '-i', 'pipe:5',
   // Rescale the video
   '-vf', 'scale=320:240',
   // Choose some fancy codes
   '-c:v', 'libx265', '-x265-params', 'log-level=0',
   '-c:a', 'flac',
   // Define output container
-  '-f', 'matroska', 'pipe:5',
+  '-f', 'matroska', 'pipe:6',
 ], {
   windowsHide: true,
   stdio: [
     /* Standard: stdin, stdout, stderr */
     'inherit', 'inherit', 'inherit',
-    /* Custom: pipe:3, pipe:4, pipe:5 */
-    'pipe', 'pipe', 'pipe',
+    /* Custom: pipe:3, pipe:4, pipe:5, pipe:6 */
+    'pipe', 'pipe', 'pipe', 'pipe',
   ],
 });
 ffmpegProcess.on('close', () => {
-  process.stdout.write('\n\n\n');
+  process.stdout.write('\n\n\n\n');
   clearInterval(progressbar);
   console.log('done');
 });
 
 // Link streams
 // FFmpeg creates the transformer streams and we just have to insert / read data
-audio.pipe(ffmpegProcess.stdio[3]);
-video.pipe(ffmpegProcess.stdio[4]);
-ffmpegProcess.stdio[5].pipe(fs.createWriteStream('./out.mkv'));
+ffmpegProcess.stdio[3].on('data', chunk => {
+  // Parse the param=value list returned by ffmpeg
+  const lines = chunk.toString().trim().split('\n');
+  const args = {};
+  for (const l of lines) {
+    const parts = l.trim().split('=');
+    args[parts[0]] = parts[1];
```

Comment:
change `parts` to `[key, value]` so it's clearer
```suggestion
    const [key, value] = l.trim().split('=');
    args[key] = value;
```